src/utils.ts

```diff
@@ -275,6 +436,52 @@ export function getRefillsAllowed(
   }
 }
 
+/**
+ * Given a refernece in format "MedicationReference/uuid" or just "uuid", returns just the uuid compoennt
+ */
+export function getUuidFromReference(reference: string) {
+  if (reference?.includes("/")) {
+    return reference.split("/")[1];
+  } else {
+    return reference;
+  }
+}
+
+/**
+ * Returns true/false whether the most passed in medication dispense status is the most recent
+ * @param medicationDispenses
+ */
+export function isMostRecentMedicationDispenseStatus(
+  medicationDispense: MedicationDispense,
+  medicationDispenses: Array<MedicationDispense>
+) {
+  const sorted = medicationDispenses?.sort(
+    sortMedicationDispensesByDateRecorded
+  );
+
+  // prettier-ignore
+  return medicationDispense &&
+    sorted &&
+    sorted.length > 0 &&
+    sorted[0].id === medicationDispense.id ? true : false;
+}
+
+/**
+ * Revalidated (reloads) both the prescription associated with the encounter uuid,
+ * and the entire prescrption table
+ * @param encounterUuid
+ */
+export function revalidate(encounterUuid: string) {
```

Comment:
This utility function uses the useSWR mutate function to trigger a revalidation of the SWR, so that the data is updated on any change.

Each time we make a change we need to:
1) reload all tabs in the prescription table (as some prescriptions may be added/removed from a table due to state change)
2) reload the prescriptions details of the specific prescription/encounter we are working with

Thie method provides a filter function to match and invalidate the appropriate keys.